lib/puppet/version.rb

```diff
@@ -7,7 +7,7 @@
 
 
 module Puppet
-  PUPPETVERSION = '4.7.0'
+  PUPPETVERSION = '4.7.1'
```

Comment:
Is this the correct version number? Do we bump this separately after this mergeup?